sift-cli.js

```diff
@@ -235,21 +235,13 @@ const setupSalt = async () => {
       await fs.writeFileAsync(aptSourceList, `deb http://repo.saltstack.com/py3/ubuntu/${osVersion}/amd64/${saltstackVersion} ${osCodename} main`)
       await child_process.execAsync(`wget -O - https://repo.saltstack.com/py3/ubuntu/${osVersion}/amd64/${saltstackVersion}/SALTSTACK-GPG-KEY.pub | apt-key add -`)
       await child_process.execAsync('apt-get update')
-      await child_process.execAsync('apt-get install -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" -y --allow-change-held-packages salt-minion', {
-        env: {
-          DEBIAN_FRONTEND: 'noninteractive',
```

Comment:
This one could be the error itself? I'm not quite sure how the .js is converting for bash, bash wise this should be

`DEBIAN_FRONTEND=noninteractive `